Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -252,10 +252,15 @@ private func createResolvedPackages(
         // The diagnostics location for this package.
         let diagnosticLocation = { PackageLocation.Local(name: package.name, packagePath: package.path) }
 
-        // Get all the system module dependencies in this package.
-        let systemModulesDeps = packageBuilder.dependencies
+        // Get all implicit system library dependencies in this package.
+        let implicitSystemTargetDeps = packageBuilder.dependencies
             .flatMap({ $0.targets })
-            .filter({ $0.target.type == .systemModule })
+            .filter({
```

Comment:
Could we simplify this to `($0.target as? SystemLibraryTarget)?.isImplicit ?? false`

Reply:
I wouldn't call one or the other a simplified version, heh.